proxies/venue-graphql-proxy/src/resolvers/eventResolver.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const Event = {
```

Comment:
I think, we should not need the Event in this server. It's something that was used in the old production Liikunta app, but in the monorepo we have a full connection to LE through the another proxy. This could be deleted or at least deprecated.

Reply:
I rather delete it